0.download_data/scripts/download_plates.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Download NF1 datasets from Figshare
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+import download_figshare as df
```

Comment:
`df` is confusing b/c it also is shorthand for `dataframe`. Why just keep the full name `download_figshare`? This is perfect!

Reply:
Huh, that is true! I have now changed it to:

```python
from download_figshare import download_figshare
```

I wasn't a fan of how the function looked, e.g. `download_figshare.download_figshare`, so this way looks cleaner.